doc/README.md

```diff
@@ -3,7 +3,7 @@ Bitcoin Core
 
 Setup
 ---------------------
-Bitcoin Core is the original Bitcoin client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Bitcoin transactions (which is currently more than 100 GBs); depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to a day or more.
+Bitcoin Core is the original Bitcoin client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Bitcoin transactions, which requires several hundred gigabytes of disk space. Depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to a day or more.
```

Comment:
Could say that currently grows around 50GB/year?

Reply:
I don't htink that's very useful here

Reply:
This doc is not a guide for running a node, but a more general text. As such, it does not need to be specific about the storage reqs, in my opinion. This passage would benefit from a general enough phrasing that won't need to be updated for a long time. Even the text "several hundred gigabytes" is not future proof, but that may be an ok tradeoff. 
If we find a good and independent reference source for the size further on, we can add a link to it then.